### PR TITLE
Document that testable representations of all pipeline contexts is provided

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: nservicebus/pipeline/index.md [hunk 1]
```diff
@@ -12,18 +12,20 @@ related:
...
+* [Aborting the Pipeline](/nservicebus/pipeline/aborting.md)
+* [Reacting to pipeline events](/nservicebus/pipeline/events.md)
 
-### Features build on the pipeline
+ Unit testing of custom extensions is supported by [the `NServiceBus.Testing` library](/nservicebus/testing/#testing-a-behavior).
 
- * [DataBus](/nservicebus/messaging/databus/)
- * [Message Property Encryption](/nservicebus/security/property-encryption.md)
- * [Recoverability](/nservicebus/recoverability/)
+## Features build on the pipeline
```
Comment (bording) [inline nservicebus/pipeline/index.md]: Should be "built" not "build"

File: nservicebus/testing/index_content_testing_[6,).partial.md [hunk 2]
```diff
@@ -29,4 +26,6 @@ snippet: SampleBehavior
 
 The behavior can be tested similar to a message handler or a Saga by using a testable representation of the context:
 
-snippet: BehaviorTest
\ No newline at end of file
+snippet: BehaviorTest
+
+Testable representations are provided for all pipeline behaviour contexts, see [the pipeline documentation](/nservicebus/pipeline/) for further details.
```
Comment (bording) [inline nservicebus/testing/index_content_testing_[6,).partial.md]: Should be "behavior", not "behaviour".

Comma should be changed to a period, "see" should be "See"

File: nservicebus/testing/index_content_testing_[6,).partial.md [hunk 3]
```diff
@@ -20,7 +18,6 @@ The following unit tests checks that the total amount has the discount applied:
 
 snippet: SagaTest
 
-
 ### Testing a behavior
```
Comment (timbussmann) [inline nservicebus/testing/index_content_testing_[6,).partial.md]: should this be moved one level up too then?

**Comment by andreasohlund** [comment]
@timbussmann @bording addressed your comments.

Thoughts on perhaps adding a bullet list of all the testable context and what they mock to improve the searchability [mentioned by @SimonCropp ](https://github.com/Particular/docs.particular.net/issues/3971#issuecomment-413355797)?

**Comment by andreasohlund** [comment]
In that case lets get this PR in. We can always add the list later on

**Pull request merged**